src/common/address.c

```diff
@@ -1762,75 +1820,27 @@ get_interface_address6_via_udp_socket_hack,(int severity,
   return r;
 }
 
-/** Set *<b>addr</b> to an arbitrary IP address (if any) of an interface that
```

Comment:
Please don't move functions, it makes them harder to review.

If you want to move functions, do it in a separate commit, and make that commit the first or last commit of the branch. (On GitHub, please make a separate pull request.)